### ISSUE TITLE
twiddler.addGroup

### ISSUE THREAD

**Issue by missinglink** [mnaberez/supervisor_twiddler#2]
The function twiddler.addGroup is not available.

The examples given in the README file do not match the current state of the twiddler master branch.

Maybe this commit is missing?

https://github.com/waptang/supervisor_twiddler/commit/ab76aacde2fbea653ad7dd64e61eab141990f8fe

**Comment by Sunzhifeng** [comment]
Yes,  one more request for the AddGroup() , that I need to manage my process group dynamicly. Thx so much
